matterpoll/matterpoll
# First draft

### What does this PR do?
- Add a prototype for matterpoll
- Add task as build tool (Fixes #2)
- Add travis file (Fixes #3)
- Calculate Code coverage (Fixes #4)
- Use dep to manage decencies (Fixes #5)

### How to test this PR?
To test the prototype build the package with `task dist` and upload it your mattermost server. Post `/matterpoll "Question" "Answer 1" "Answer 2"` in a channel and see the answer.

### Notes
This does not the ability to update a poll by clicking an answer button. This will be done in a different PR. All code is subject to change especial output and strings.
